spec/features/course/assessment/answer/text_response_answer_spec.rb

```diff
@@ -40,13 +41,27 @@
 
           # The file should show in the dropzone
           expect(dropzone).to have_css('span', text: 'text.txt')
-
-          click_button('Save Draft')
+          wait_for_job
```

Comment:
What job? any job invoked?

Reply:
the intention is actually to test for the upload file autosave. Basically putting the file into the dropzone is supposed to trigger the autosave automatically.

However, this test is considered flaky by the system. Will find another way to trigger the autosave for the file upload feature

Reply:
Then, dont use wait_for_job. wait_for_job is used to wait for job in the backend. If you really need to wait, then create another function (wait_for_autosave that contains sleep). If the content of wait_for_job is changed next time, this test would not break!

Reply:
Done